package.json

```diff
@@ -31,6 +31,7 @@
     "precommit": "lint-staged",
     "setup": "yarn run build",
     "test": "yarn run jest",
+    "type": "nimbus typescript --build --reference-workspaces",
```

Comment:
Should only emit declarations, otherwise the TS build would overwrite the Babel build.

Reply:
🤦‍♂ done

Reply:
oops not compatible with `--reference-workspaces`